DQM/L1TMonitor/python/L1TStage2uGT_cfi.py

```diff
@@ -1,9 +1,11 @@
 import FWCore.ParameterSet.Config as cms
-
-l1tStage2uGT = cms.EDAnalyzer("L1TStage2uGT",
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
```

Comment:
@dtsitson  This part of the changes does not work for 10_0_X as discussed yesterday. Please revert it back to the previous state.